### ISSUE TITLE
Printing of sign corrections is broken

### ISSUE THREAD

**Issue by haudren** [Jmeyer1292/opw_kinematics#15]
I am testing this package on my machine, and I noticed that printing of the sign corrections is broken. This is my test example:
> Distances: [0.5 0.5 0.5 0.5 0.5 0.5 0.5]
> Offsets = [0 1 2 3 4 5 ]
> Sign_corrections = [ �  �  � ]

Sign corrections should contain alternating `1` and `-1`.

A simple fix is to add a cast:
```diff
-os << params.sign_corrections[i] << " ";
+os << (int)params.sign_corrections[i] << " ";
```

But I guess I rather wonder why you went with `signed char` instead of int?

**Comment by Jmeyer1292** [comment]
@haudren We just figured smaller was better since all we were doing is storing a sign. You have indeed pointed out a bug with the printing.

**Comment by gavanderhoorn** [comment]
Should be fixed with the merge of #17.
